pkgs/by-name/yp/ypbind-mt/package.nix

```diff
@@ -0,0 +1,74 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  autoreconfHook,
+  docbook-xsl-nons,
+  libnsl,
+  libtirpc,
+  libxcrypt,
+  libxml2,
+  libxslt,
+  pkg-config,
+  rpcbind,
+  systemdLibs,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ypbind-mt";
+  version = "2.7.2";
+
+  nativeBuildInputs = [
+    autoreconfHook
+    docbook-xsl-nons
+    pkg-config
+    libxml2
+    libxslt
+  ];
+
+  buildInputs = [
+    libnsl
+    libtirpc
+    libxcrypt
+    rpcbind
+    systemdLibs
+  ];
+
+  src = fetchFromGitHub {
```

Comment:
usually src comes after version

Reply:
Has been fixed in [`8e49884` (#333099)](https://github.com/NixOS/nixpkgs/pull/333099/commits/8e498848af210a5a743e44fbfd5ac4f28d657969)